spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/subscriber/PubSubSubscriberOperations.java

```diff
@@ -114,27 +115,32 @@
 	PubsubMessage pullNext(String subscription);
 
 	/**
-	 * Acknowledge a batch of messages. The method will group the messages by subscription name
-	 * and acknowledge them in batches.
+	 * Acknowledge a batch of messages. The messages must have the same project id and be from the same subscription.
 	 * @param acknowledgeablePubsubMessages messages to be acknowledged
+	 * @return {@code ListenableFuture<Void>} the ListenableFuture for the asynchronous execution
+	 * @since 1.1
```

Comment:
I don't think putting `@since 1.1` is correct here, given that the method was available in the `1.0` release, granted with a `void` return, but still.

Reply:
Fixed.